src/app/edit-create-study-set/edit-create-study-set.component.ts

```diff
@@ -36,14 +37,15 @@ import { RouteParamNotFound } from '../errors/route-param-error';
   styleUrl: './edit-create-study-set.component.scss'
 })
 export class EditCreateStudySetComponent {
-  @Input() userId: string = "no Id passed"; // remove later
+  @Input() userId: string = sessionStorage.getItem("uid")!; // remove later
```

Comment:
remove the "remove later" comment because this is probably the finalized version of this code 😄